content/plotly/concepts/express/terms/pie/pie.md

```diff
@@ -0,0 +1,48 @@
+# Plotly Pie Charts
+
+## .pie()
+
+In python, we can use *.pie()* from the *plotly library* to display data in the form of a pie chart.
+This is such a simple tool to use and can transform data into visually apealing ways for readers
+to get a better idea about the relationships between variables.
```

Comment:
```suggestion
In Python, the **`pie()`** method from the Plotly library is used to create a pie chart, which visually represents data proportions. This method helps transform data into an appealing visual format, making it easier for readers to understand the relationships between variables.
```

Reply:
Reframed the sentence for better clarity. 
- Avoid using "we", "you" in the sentences.
- The first instance of the term should be in bold and backticks: **`.pie()`**